qemu/tests/cfg/hpt.cfg

```diff
@@ -0,0 +1,36 @@
+# Notes:
+#     These test cases are only for HPT guest,which include Power8 guest
+#     and Power8 guest with compatible mode running on Power9 host.
+- hpt:
+    type = hpt_miscellaneous
+    no Host_RHEL.6
+    no RHEL.5 RHEL.6
+    only pseries
+    variants:
+        - increase:
+            mistype = "increase"
+            increment = "1"
+        - increase-reboot:
+            mistype = "increase-reboot"
+            increment = "1"
+        - reduce:
+            mistype = "reduce"
+            increment = "-1 2 -1"
+        - reduce-reboot:
+            mistype = "reduce-reboot"
+            increment = "-1 2 -1"
+        - mem_hotplug:
+            mistype = "mem"
+            increment = "1 -1"
+            monitors = qmp1
+            monitor_type = qmp
+            main_monitor = qmp1
```

Comment:
What is the reason to customize `monitors`?

Reply:
Remove those parameters since finding another method,thanks